thoth/storages/data/alembic/versions/dd19dd789f6d_added_image_size_attribute_to_.py

```diff
@@ -0,0 +1,28 @@
+"""Added image_size attribute to PackageExtractRun
+
+Revision ID: dd19dd789f6d
+Revises: bcf07d69fc15
+Create Date: 2019-10-25 21:21:49.504320+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'dd19dd789f6d'
+down_revision = 'bcf07d69fc15'
```

Comment:
Please update version, this will introduce wrong head to alembic migrations. The current most recent version is a05ad6e1d0ca introduced in c52f31b.

Reply:
Is this solved?

Reply:
I changed!